vdr/didsubject/did_document_test.go

```diff
@@ -76,18 +79,22 @@ func TestSqlDIDDocumentManager_CreateOrUpdate(t *testing.T) {
 	t.Run("update", func(t *testing.T) {
 		tx := db.Begin()
 		docManager := NewDIDDocumentManager(tx)
-		_, err := docManager.CreateOrUpdate(sqlDidBob, []orm.VerificationMethod{vm}, []orm.Service{service})
+		docRoot, err := docManager.CreateOrUpdate(sqlDidBob, []orm.VerificationMethod{vm}, []orm.Service{service})
 		require.NoError(t, err)
 		require.NoError(t, tx.Commit().Error)
 
 		docManager = NewDIDDocumentManager(transaction(t, db))
 		require.NoError(t, err)
+		time.Sleep(time.Second) // make sure doc.CreatedAt < doc.UpdatedAt
```

Comment:
try to prevent this. Could be done by setting a timeNow func in the package.

Reply:
but using gorm autoCreate makes this totally impossible...

Reply:
you could manually adjust the field value in the DB (low level) and then call update